mne/channels/_standard_montage_utils.py

```diff
@@ -254,15 +235,19 @@ def _read_elc(fname, head_size):
                             nasion=nasion, lpa=lpa, rpa=rpa)
 
 
-def _read_theta_phi_in_degrees(fname, head_size, fid_names):
-    options = dict(skip_header=1, dtype=(_str, 'i4', 'i4'))
-    ch_names, theta, phi = _safe_np_loadtxt(fname, **options)
+def _read_theta_phi_in_degrees(fname, head_size, fid_names=None,
+                               add_fiducials=False):
+    ch_names, theta, phi = _safe_np_loadtxt(fname, skip_header=1,
+                                            dtype=(_str, 'i4', 'i4'))
+    if add_fiducials:
+        assert fid_names is None
+        fid_names = ['Nasion', 'LPA', 'RPA']
+        ch_names.extend(fid_names)
+        theta = np.append(theta, [115, -115, 115])
```

Comment:
Can you add a comment where you got this definition for nasion/lpa/rpa in spherical coords? It will help the next time we think about this

Reply:
Added a comment and change log.